src/NuGet.Server.V2.OWinSampleHost/Controllers/NuGetPublicODataController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NuGet.Server.Core.Infrastructure;
+using NuGet.Server.V2.Controllers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+
+namespace NuGet.Server.V2.OWinSampleHost
+{
+    public class NuGetPublicODataController : NuGetODataController
+    {
+        public NuGetPublicODataController()
+            : base(Program.NuGetPublicRepository, new ApiKeyPackageAuthenticationService(true,Program.ApiKey))
+        {
+
+        }
+
+        public override HttpResponseMessage Download(string id, string version = "")
```

Comment:
Not needed. Just meant to show someone browsing the samples that methods from the NuGetODataController can be overriden in their own implementation.